### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by StrangeDOS** [magnuschase293/comic_getter#32] [Ideas]
This would be extremely useful...to me and probably no one else.  lol

I would be great if we had a way to scrape the entire list (or a filtered list) of all the comics on the site.  

Example not using filters: Tool goes out the the site and starts gathering this list
![image](https://user-images.githubusercontent.com/20542548/139560150-83e108ba-ffee-4076-b6ec-328edab748d0.png)
...and it could export to an Excel sheet.
![image](https://user-images.githubusercontent.com/20542548/139560172-07a3e44a-e6ed-489a-8e1f-d098360a7709.png)
NOTE:  The items in column A actually hold two bits of info.  First the title of the comic, and second, the URL.  You could seperate out those items and have a third column that has the URL.

You could use it to capture the ENTIRE list which could be reviewed offline.

Example using filters:  The tool allows you to use keywords.  So if I added filters for "Superman", "Spider-Man", "Ultimates".  Then it will collect the same type of list as the first example but only populating the list with titles and URLs that contain one of the keywords.  

NOTE:  The info in column B is handy as I don't like to download things until they are complete or at a high number of issues.

Anyway, that's my pitch.

**Comment by magnuschase293** [comment]
Very interesting idea, I once pondered doing something similar in issue #7, but the fact that I thougt of using a JSON and that the console isn't exactly ideal for showing multiple possible matches made me discard the idea. 

Your suggestion has much more promise. The fact that Excel has a clean interface and lots of sorting algorithms built-in is a huge plus. A few ideas that come to my mind that may make this even better are:

* The links should definitely be on a different column in order for them to be easily selectable.
* Scrape a bit of extra info found by just hovering over comic names in Read Comic Online list.
* Use some VBA to run the script directly from within excel and for instance select a couple of cells.
* A button within the spreadsheet to update it.

Have in mind most of this extra features need further research to see how doable they are and that it will probably take me around three to four months to actually have it done since during November and December I am going to be focused on other stuff.

**Reply by StrangeDOS** [reply]
Bringing it into Excel really opens it up for Excel users like me.  I have to do an insane amount of scripting to manage my download list and then for sorting.  So that would really work for me.  I look forward to seeing what you come up with.  But I understand about how busy end of year is.
